src/Network/Network/help/Get-AzVirtualNetworkGatewayVpnclientConnectionHealth.md

```diff
@@ -0,0 +1,146 @@
+﻿---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/az.network/get-azvirtualnetworkgatewayvpnclientconnectionhealth
+schema: 2.0.0
+---
+
+# Get-AzVirtualNetworkGatewayVpnclientConnectionHealth
+
+## SYNOPSIS
+Get the list of vpn client connection health of an Azure virtual network gateway for per vpn client connection
+
+## SYNTAX
+
+```
+Get-AzVirtualNetworkGatewayVpnclientConnectionHealth -VirtualNetworkGatewayName <String> -ResourceGroupName <String> -InputObject <PSVirtualNetworkGateway> -ResourceId <ResourceId>
```

Comment:
It looks like documentation wasn't updated from when the name became Get-AzVirtualNetworkGatewayVpn**C**lientConnectionHealth. Could you regenerate it?